images/01-base/Dockerfile

```diff
@@ -35,6 +35,9 @@ COPY inputrc /etc/inputrc
 COPY growpart /usr/bin/growpart
 RUN sed -i s/"partx --update \"\$part\" \"\$dev\""/"partx --update --nr \"\$part\" \"\$dev\""/g /usr/bin/growpart && \
     sed -i -e 's/duid/clientid/g' /etc/dhcpcd.conf && \
-    sed -i 1,10d /etc/rsyslog.conf
+    sed -i 1,10d /etc/rsyslog.conf && \
+    echo "*.*                /var/log/syslog" >> /etc/rsyslog.conf
```

Comment:
I'm a little confused what these few changes are for. Would you mind explaining the purpose here? :confused: